main.js

```diff
@@ -1,9 +1,18 @@
 const fs = require('fs');
+const path = require('path');
+
+const folderPath = 'testData';
+const pathFiles = fs.readdirSync(folderPath).map(fileName => path.join(folderPath, fileName));
+const textsForAnalysis = pathFiles.map(pathToFile => fs.readFileSync(pathToFile, 'utf-8'));
+
+function readFiles(folder) {
+  folder.forEach(function (arrayElement) {
+    const textExample = fs.readFileSync(arrayElement, 'utf-8');
+    textsForAnalysis.push(textExample);
+    return textsForAnalysis;
+  });
+}
```

Comment:
Please have next code here:

```
function readFiles(folderPath) {
  const pathFiles = fs.readdirSync(folderPath).map(fileName => path.join(folderPath, fileName));
  return pathFiles.map(pathToFile => fs.readFileSync(pathToFile, 'utf-8'));
}
```

and then modify last line in the file:

```
console.log(calculateWordsStatistic(readFiles('testData')));
```

Reply:
changes made